filesystem/btrfs_stream.ksy

```diff
@@ -0,0 +1,151 @@
+meta:
+  id: btrfs_stream
+  application: Btrfs
+  file-extension: btrfs-stream
+  endian: le
+  license: CC0-1.0
+doc: |
+  This spec can be used to disassemble raw Btrfs Send stream format
+  version 1 from the output of `btrfs send` command.
+  See also the `btrfs receive --dump` command.
+doc-ref: https://btrfs.wiki.kernel.org/index.php/Design_notes_on_Send/Receive
+
+seq:
+  - id: header
+    type: send_stream_header
+  - id: commands
+    type: send_command
+    repeat: eos
+types:
+  send_stream_header:
+    seq:
+      - id: magic
+        contents: ['btrfs-stream', 0x00]
+      - id: version
+        type: u4le
+  send_command:
+    seq:
+      - id: size
+        type: u4le
+      - id: type
+        type: u2le
+        enum: command
+      - id: checksum
+        size: 4
+      - id: data
+        type: trios_of_type_length_value
+        size: size
+        #if: size > 0
+    types:
+      trios_of_type_length_value:
+        seq:
+          - id: tlv
+            type: tlv
+            repeat: eos
```

Comment:
More common practice, when there is a container type for values of type `foo` to name that container `foo` in plural, i.e. `foos`:

```suggestion
      - id: data
        type: tlvs
        size: size
        #if: size > 0
    types:
      tlvs:
        seq:
          - id: tlv
            type: tlv
            repeat: eos
```

Reply:
Renamed trios_of_type_length_value to tlvs